lib/BridgeAbstract.php

```diff
@@ -8,13 +8,26 @@
 	const MAINTAINER = 'No maintainer';
 	const CACHE_TIMEOUT = 3600;
 	const PARAMETERS = array();
+	const FORMATS = array(); // array of compatibles formats or null
 
 	protected $cache;
+	protected $extraInfos;
 	protected $items = array();
 	protected $inputs = array();
 	protected $queriedContext = '';
 
 	/**
+	* Return cachable datas (extrainfos and items) stored in the bridge
+	* @return mixed
+	*/
+	public function getCachable(){
+		$cachable = array();
+		$cachable["items"] = $this->getItems();
+		$cachable["extraInfos"] = $this->getExtraInfos();
+		return $cachable;
```

Comment:
Instead of creating an array every time, this kind of syntax would be lighter :
```PHP
return array('items' => $this->getItems(),  'extraInfos' => $this->getExtraInfos());
```